rootfs/etc/cont-init.d/pulse-config.sh

```diff
@@ -6,7 +6,7 @@ declare tsched
 readonly ALSA_CARDS="$(aplay -l)"
 
 # RaspberryPi
-if echo "${ALSA_CARDS}" | grep -q "\[bcm2835 .*\]"; then
+if [[ "${ALSA_CARDS}" =~ \[bcm2835 .*\] ]]; then
```

Comment:
```suggestion
if [[ "${ALSA_CARDS}" =~ "\[bcm2835 .*\]" ]]; then
```

Reply:
why it needs`"`? Is that the best praxis?

Reply:
I see